### PR TITLE
networking: Add MTU value test for SRIOV NAD's

### PR DESCRIPTION
**Pull request by sebrandon1**
Adds a test that checks that Network Attachment Definitions are specifying their MTU values explicitly.

EDIT: I was trying to follow the quick start guide [here](https://github.com/openshift/sriov-network-operator/blob/master/doc/quickstart.md) for the SRIOV network operator but I don't have access to the necessary [hardware](https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/doc/supported-hardware.md).

We'll have to maybe figure out how to test this via QE on supported hardware in the future.

### CODE REVIEW

File: pkg/provider/pods.go [hunk 1]
```diff
@@ -265,6 +265,61 @@ func getCNCFNetworksNamesFromPodAnnotation(networksAnnotation string) []string {
 	return networkNames
 }
 
+// isNetworkAttachmentDefinitionSRIOVConfigMTUSet is a helper function to check whether a CNI config
```
Comment (bnshr) [inline pkg/provider/pods.go:268]: Remove this commented code.

**Review by greyerof — APPROVED** [review]

File: pkg/provider/pods.go (hunk 1 above)
Comment (sebrandon1) [reply pkg/provider/pods.go:268]: Actually this was on purpose.  This is also done on other functions in this `pods.go` file to show examples of the JSON that is being parsed.

**Review by bnshr — APPROVED** [review]

**Comment by bnshr** [comment]
Should there be a QE test for this which can be skipped based on the unavailability of the hardware?

**Comment by sebrandon1** [comment]
@bnshr Yes, that would be the goal for this test I'd say.  We'll need some sort of hardware environment to test it.

**Pull request merged**